src/tutorials/0005/04.md

```diff
@@ -22,16 +22,4 @@ If we knew the path to a file in IPFS we wanted to read, we could call the `cat`
 await ipfs.cat('QmWCscor6qWPdx53zEQmZvQvuWQYxx1ARRCXwYVE4s9wzJ')
 ```
 
-
-The IPFS path can take several formats (you can read about them in the [File API](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md#cat) documentation). In this lesson, we will use the simplest one: the `hash` string present in the objects we receive when the `ipfs.add` function is called.
-
-The returned data comes in the form of an `Buffer`. If you uploaded a plain text file (such as a `.txt`), you can read the contents of the file by calling `result.toString('utf8')`.
```

Comment:
Isn't it important to explain the `toString` method in the lesson's body?

Reply:
It's still there. I was removing duplicate contents after some sort of failed merge situation. :) Just further fixed my original edits in a new commit to replace a missing single-line example.